src/boot/efi/boot.c

```diff
@@ -103,7 +103,16 @@ typedef struct {
 
 /* These values have been chosen so that the transitions the user sees could
  * employ unsigned over-/underflow like this:
- * efivar unset ↔ force menu ↔ no timeout/skip menu ↔ 1 s ↔ 2 s ↔ … */
+ * efivar unset ↔ force menu ↔ no timeout/skip menu ↔ 1 s ↔ 2 s ↔ …
+ *
+ * Note: all the values below are ABI, so they are not allowed to change.
+ * The bootctl tool sets the numerical value of TIMEOUT_MENU_FORCE and
+ * TIMEOUT_MENU_HIDDEN, instead of the string for compatibility reasons.
+ *
+ * The other values may be set by systemd-boot itself and changing those
+ * will lead to functional regression when new version of systemd-boot is
+ * installed.
```

Comment:
btw, we generally break at 109ch (some old code uses other stuff, but please rebreak any comments you see to 109ch)

.editorconfig, and the vim/emacs counterparts are all configured to 109 automatically